Problem1.3/cal_Schwarzchild_radius.py

```diff
@@ -0,0 +1,15 @@
+"Problem1.3(a) Write a python script, using pint, which finds the Schwarzchild radius of the Sun, in m."
+import numpy as np
+import pint
+from astropy import constants as const
+
+units=pint.UnitRegistry()
+def Cal_R_Sch(m_in_solarms):
+    const_G=const.G.value*units.meter**3/(units.kilogram*units.second**2)
+    const_solarmass=const.M_sun.value*units.kilogram
+    const_sol=const.c.value*units.meter/units.second
+
+    return 2*const_G*const_solarmass*m_in_solarms/const_sol**2
```

Comment:
You can make it do the work for you easier with:
const_G = units.newtonian_constant_of_gravitation
and const_c = units.speed_of_light
then it will convert automatically if you use R_s = ().to('m')